src/Tribe/Query.php

```diff
@@ -51,6 +51,13 @@ public static function can_inject_date_field( $query ) {
 				$can_inject = false;
 			}
 
+			/**
+			 * When using the ORM we cannot use EventStartDate injection
+			 */
+			if ( isset( $query->builder ) && $query->builder instanceof Tribe__Repository ) {
```

Comment:
If `$query` is an object the `isset` is not required as `instanceof` will return `false` if is not defined.

Reply:
On this case the `builder` param might now always be set, since we are relying on maybe been a normal WP_Query.

Reply:
I see but `$query` is defined right, which in this case `$query->builder` is `null` and in that case `instanceof` return `false`

Reply:
What I meant, is that `$query` is based on `WP_Query` which doesn't by definition have a `builder` param, which would lead to a notice of trying to fetch an un-existent param on the object.

Reply:
sounds good